EarTrumpet/App.xaml.cs

```diff
@@ -247,5 +249,33 @@ private SettingsCategoryViewModel CreateAddonSettingsPage(IEarTrumpetAddonSettin
         }
 
         private Window CreateMixerExperience() => new FullWindow { DataContext = new FullWindowViewModel(CollectionViewModel) };
+
+
+        private void AbsoluteVolumeIncrement()
+        {
+            foreach (var device in CollectionViewModel.AllDevices.Where(d => !d.IsMuted || d.absMuted))
```

Comment:
Nit: PascalCase public properties (and start with Is/Has)
```suggestion
            foreach (var device in CollectionViewModel.AllDevices.Where(d => !d.IsMuted || d.IsAbsMuted))
```